Math/NumberTheory/GaussianIntegers.hs

```diff
@@ -138,7 +138,7 @@ primes = [ g
                 else
                     if p == 2
                     then [1 :+ 1]
-                    else let x :+ y = findPrime' p
+                    else let x :+ y = findPrime' (fromMaybeError "Impossible error" $ toFieldCharacteristic p) -- safe actually
```

Comment:
In future `Sieve.primes` will return `[Prime Integer]`, but in meantime we can export `Math.NumberTheory.Primes.Types` and use `FieldCharacteristic (PrimeNat p) 1` directly.